doc/src/ase_userguide/ase_userguide.md

```diff
@@ -670,6 +670,10 @@ is set to ```/tmp/opae/ase/work```.  Then, start the software application.
     $ export LD_LIBRARY_PATH=/tmp/opae/myinst/lib/
     $ LD_PRELOAD=libopae-c-ase.so ./hello_fpga
 
+    # Run the application through with ```with_ase``` instead of ```LD_PRELOAD```
```

Comment:
Both "LD_PRELOAD=libopae-c-ase.so ./hello_fpga"  and  "with_ase ./hello_fpga" should work. Could you change the comment here?